examples/hello_world/disagg_skeleton/components/worker.py

```
@@ -109,7 +156,8 @@ async def worker_generate(self, request: GeneralRequest):
             callback = self.get_remote_prefill_request_callback()
             await callback(prefill_request)
 
-        print(f"{self.hostname}: Worker invoked")
+        logger.info(f"{self.hostname}: Worker invoked")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)